### DISCUSSION TITLE
Options bar (taskbar) not showing up

### DISCUSSION THREAD

**Discussion by BENZOOgataga** [pear-devs/pear-desktop#358] [Q&A] [answered]
To make it short i clicked somewhere on the taskbar and now it doesn't show up and of course i can't disable the option i clicked on because obviously the taskbar isn't there anymore. What i mean by "taskbar" is the options bar on the top of the app, the bar that is show by default. (Where you enable adblocker for example)

I tried to reinstall the app or even upgrade to 1.12.1 or 1.12.2 (pre-release) to fix this but nothing happens, i'm running on Windows 10. Any ideas ?

**Comment by Araxeus** [comment, marked as answer]
click `alt` or `escape` to show the menu
you probably enabled the `auto-hide-menu` option

**Reply by BENZOOgataga** [reply]
Ok thanks it's working again !

**Reply by AivinKoshy** [reply]
i also had auto hide and the menu bar used to show up when i press `alt`  , but now its not working for me.

**Reply by inson1** [reply]
@WhiskeyMikeRomeo Did you solve it?
